igcollect/artfiles.py

```diff
@@ -1,12 +1,18 @@
-#!/usr/bin/env python
```

Comment:
Why did you remove this line? PEP8 says keep python if it is python2 and python3.

Reply:
I removed it because the script is executed by explicitly calling the Python binary. `/usr/bin/python3 artfiles.py`.

But looking at PEP8, it should be there in the main file, so I'll add it back.